liquidcore/home/management/commands/createusers.py

```diff
@@ -1,5 +1,5 @@
 import json
-from pathlib import Path
+from pathlib import Path  # noqa: F401
```

Comment:
You can totally delete the import here, since it has no way of being used. You only want to ignore this error when you import something in a `try except` or except it to have side effects.